packages/react-instantsearch/src/connectors/connectScrollTo.js

```diff
@@ -1,15 +1,21 @@
 import PropTypes from 'prop-types';
 
 import createConnector from '../core/createConnector';
-import { getCurrentRefinementValue } from '../core/indexUtils';
-
+import {
+  getCurrentRefinementValue,
+  hasMultipleIndex,
+  getIndex,
+} from '../core/indexUtils';
+import { shallowEqual } from '../core/utils';
+import { omit } from 'lodash';
 /**
  * connectScrollTo connector provides the logic to build a widget that will
  * let the page scroll to a certain point.
  * @name connectScrollTo
  * @kind connector
  * @propType {string} [scrollOn="page"] - Widget searchState key on which to listen for changes, default to the pagination widget.
  * @providedPropType {any} value - the current refinement applied to the widget listened by scrollTo
+ * @providedPropType {boolean} hasNotChanged - indicate if the refinement came from the scrollOn observed widget
```

Comment:
indicates whether the refinement came from the scrollOn argument (for instance page if we used scrollOn(page))